space-cli/cmd/modules/auth/delete.go

```diff
@@ -0,0 +1,74 @@
+package auth
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/input"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/transport"
+)
+
+func deleteAuthProvider(project, prefix string) error {
+
+	url := fmt.Sprintf("/v1/config/projects/%s/user-management/provider", project)
+
+	// Get the spec from the server
+	payload := new(model.Response)
+	if err := transport.Client.Get(http.MethodGet, url, map[string]string{"id": "*"}, payload); err != nil {
+		return err
+	}
+
+	doesProviderExist := false
+	providers := []string{}
+	for _, item := range payload.Result {
+		spec := item.(map[string]interface{})
+		providers = append(providers, spec["id"].(string))
+	}
```

Comment:
Try to use the existing functions, GetAuthProviders which outputs spec object

Reply:
But that is excess code of creating spec objects when only the id is required. Should I still use that function?